### ISSUE TITLE
Just a question: using ODPI with client static library. How?

### ISSUE THREAD

**Issue by superriva** [oracle/odpi#77]
Official Oracle documentation has an information how to build a client static library. You could see that in the link: https://docs.oracle.com/database/121/LACLI/post_inst_task.htm#LACLI1401 paragraph 4.2.6
So, it's possible to make libclntst12.a (a client static library for oracle 12c for an example).
It will be good ODPI can compile with a client static library, because no need to install a client dynamic library. Is it possible?
Thank you.

**Comment by anthony-tuininga** [comment]
In theory it should be possible. When I get a chance I'll give it a try and let you know -- but if you try yourself, let me know what you discover!

**Comment by ddevienne** [comment]
Is that even possible on Windows? I've never heard one could avoid oci.dll. Even needs it's own VC C/C++ runtime separate from our own app.

**Comment by anthony-tuininga** [comment]
I don't think it is possible on Windows but you can at least dump all of the appropriate DLLs in the same directory as your ODPI-C DLL (or ODPI-C based application) which functions similarly.

**Comment by cjbj** [comment]
@superriva the doc you quoted is obsolete and I will get it fixed.  The official word is in https://docs.oracle.com/database/121/LADBN/toc.htm#LADBN118

> You must use the dynamic Oracle client libraries to link the client code on Linux. Do not link the static Oracle client libraries.

**Comment by cjbj** [comment]
Looks like it was already removed.  I don't see it in https://docs.oracle.com/en/database/oracle/oracle-database/18/lacli/recommended-postinstallation-tasks.html#GUID-01E7E498-DC11-411E-ADA2-02C2B8F358F0
